benchmarks/edge_profiles.py

```diff
@@ -17,18 +17,18 @@ def fill_ggd(edge_profiles, times):
     """Fill nested arrays of structures in grids_ggd and ggd substructures.
 
     Args:
-        edge_profiles: edge_profiles IDS object (either from IMASPy or AL HLI)
+        edge_profiles: edge_profiles IDS object (either from imas-python or AL HLI)
         times: time values to fill
     """
     edge_profiles.ids_properties.homogeneous_time = (
-        imaspy.ids_defs.IDS_TIME_MODE_HETEROGENEOUS
+        imas.ids_defs.IDS_TIME_MODE_HETEROGENEOUS
     )
-    edge_profiles.ids_properties.comment = "Generated for IMASPy benchmark suite"
+    edge_profiles.ids_properties.comment = "Generated for imas-python benchmark suite"
     edge_profiles.ids_properties.creation_date = datetime.date.today().isoformat()
-    edge_profiles.code.name = "IMASPy ASV benchmark"
-    edge_profiles.code.version = imaspy.__version__
+    edge_profiles.code.name = "imas-python ASV benchmark"
+    edge_profiles.code.version = imas.__version__
     edge_profiles.code.repository = (
-        "https://git.iter.org/projects/IMAS/repos/imaspy/browse"
+        "https://github.com/iterorganization/imas-python"
```

Comment:
Here and for other repo references
```suggestion
        "https://github.com/iterorganization/IMAS-Python"
```

Reply:
Updated in https://github.com/iterorganization/IMAS-Python/tree/rename_imaspy_sources_to_imas